style.css

```diff
@@ -0,0 +1,240 @@
+ *
+ * GENERIC
+ * Box-sizing...........Better default `box-sizing`.
+ *
+ * BASE
+ * Headings.............H1–H6 styles.
+ *
+ * COMPONENTS
+ * #header............The main page header.
+ * #footer............The main page footer.
+ * buttons..............Button elements.
+ *
+ * SECTIONS
+ *
+ * MODIFIER CLASSES
+ *
+ */
+
+
+
+
+
+ /*------------------------------------*\
+  SETTINGS
+\*------------------------------------*/
+
+@font-face {
+    font-family: 'aileronbold';
+    src: url('assets/fonts/Aileron-Bold-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'aileronbold';
+    src: url('assets/fonts/Aileron-Light-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'aileronbold';
+    src: url('assets/fonts/Aileron-SemiBold-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+
+/* GLOBAL VARIABLES */
+
+:root {
+
+  /* Colors */
+  --color-primary: #66C3FF;
+  --color-secondary: #363732;
+  --color-tertiary: #DCE1E9;
+
+}
+
+
+/*------------------------------------*\
+  GENERIC
+\*------------------------------------*/
+
+* {
+  box-sizing: border-box;
+}
+
+
+
+
+/*------------------------------------*\
+  BASE
+\*------------------------------------*/
+
+body {
+  font-family: aileronbold, 'Sans Serif';
+  margin: 0;
+  padding: 0;
+}
+
+
+
+
+
+/*------------------------------------*\
+  #HEADER
+\*------------------------------------*/
+
+
+/* Mobile-first styling */
+
+#header-main {
+  background-color: #000;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  color: #fff;
+  padding: 0 0.8rem;
+}
+
+.brand-title {
+  display: flex;
+  font-size: 1.5rem;
+  padding: 1rem 0;
+}
+
+.brand-logo {
+  height: 40px;
+  width: 40px;
+}
+
+.brand-name {
+  padding: 6px 0 0 8px;
+}
+
+.brand-logo__img {
+  width: 100%;
+}
+
+.navbar-links {
+  width: 100%;
+}
+
+.navbar-links ul {
+    margin: 0;
+    padding: 1rem 0;
+    width: 100%;
+    display: none;
+    flex-direction: column;
+    list-style-type: none;
+    text-align: center;
+  }
+
+.navbar-links ul.active {
+  display: flex;
+}
+
+.navbar-links li {
+  list-style: none;
+}
+
+.navbar-links li a {
+  display: block;
+  color: #fff;
+  text-decoration: none;
+  padding: 0.5rem 0;
+  cursor: pointer;
+}
+
+.navbar-links li a:hover {
+  color: var(--color-primary);
+}
+
+.toggle-btn {
+  width: 30px;
+  height: 23px;
+  position: absolute; 
+  top: 1.5rem;
+  right: 0.8rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.toggle-btn.active {
+  justify-content: space-between;
+}
+
+.toggle-btn span {
+  background-color: #fff;
+  height: 3px;
+  width: 100%;
+  border-radius: 10px;
+}
+
+
+/* Styling for devices bigger than mobile */
+
+@media (min-width: 600px) {
```

Comment:
Add another media query with the CSS syntax @media only screen and (color) for older browsers, incase some users have no access to modern browsers due to financial issues. Increasing accessibility for all users.